Merge version 1.1.0 and version 1.1.1 from Das 2208 smap l3 feature branch to main branch (#23)
nasa/harmony-opendap-subsetter: CHANGELOG.md

## v1.1.1
### 2025-01-14

> This version of HOSS releases the HOSS codebase from a feature branch that contain V1.1.0 to the main branch.

Reviewer: ```suggestion
This version of HOSS merges the feature branch containing v1.1.0 to the main branch.
```

Reviewer: ```suggestion
This version of HOSS releases the HOSS codebase from a feature branch that contains v1.1.0 to the main branch.
```

Reviewer: I don't love the sentence as structured, but at a minimum, make contain -> contains

Author: made the updates. Thanks!
